docs/swagger.yaml

```
@@ -16,19 +17,41 @@ definitions:
         type: string
       oib:
         type: string
       password:
+        minLength: 6
         type: string
       residence:
+        maxLength: 255
         type: string
       role:
+        enum:
+        - hak
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 8

(indentation)